### ISSUE TITLE
Container with filter ignores children padding

### ISSUE THREAD

**Issue by unarist** [pixijs/pixi.js#6579]
### Expected Behavior

Containers should render children's filter padding area, if the container have filters.

### Current Behavior

If the container have...

* no filters: It renders children's filter padding area
* any filters with no padding: It doesn't render children's filter padding area

### Steps to Reproduce

1. Create a sprite, and attach OutlineFilter with padding
2. Create a container, and add the sprite as children
3. Attach AlphaFilter to the container with no padding **-> outline disappears!**
4. To render outline again with AlphaFilter, you need to set padding to AlphaFilter as well as OutlineFilter

### Environment

- **`pixi.js` version**: 5.2.2
- **Browser & Version**: Chrome 80
- **OS & Version**: Windows 10 (build 18362)
- **Running Example**: https://jsfiddle.net/unarist/u9p3rfh1/10/

![image](https://user-images.githubusercontent.com/705555/79986558-37ff2000-84e7-11ea-9a95-0d12daa7265e.png)

**Comment by bigtimebuddy** [comment]
I checked and the behavior is the same in 5.2.1, 5.1.5 and dev. Probably almost all versions, but I didn't check.

Basically you need to add the same amount of padding to AlphaFilter `alpha.padding = 20`, which, I know, is a pain and this _should_ probably be done automatically... but unfortunately it's not.

In order to achieve automatic detection of children's filter padding, I believe we need the concept of `renderBounds` that is distinct from the normal `bounds`. This would include additional padding applied with filters to compute the bounds needed to render. #6306 attempted to fix but that approach was a breaking change because it altered the `bounds` calculation. That would have caused lots of issues for people trying to position things with filters  + padding.

In summary, this is a known issue. Need some fundamental changes under the hood to address. @ivanpopelyshev or @GoodBoyDigital would you add anything or have another idea how to fix?
